styles/main.scss

```diff
@@ -1,6 +1,7 @@
 // overrides first
 $primary: #002a63;
 $secondary: #4592CF;
+$tertiary: #EAF4FF;
```

Comment:
I don't think bootstrap has a $tertiary for us to override?
```
$theme-colors: (
  "primary":    $primary,
  "secondary":  $secondary,
  "success":    $success,
  "info":       $info,
  "warning":    $warning,
  "danger":     $danger,
  "light":      $light,
  "dark":       $dark
);
```
We can still keep it if required, but I think things will be easier for us if we can stick to bootstrap conventions where possible

Reply:
Yeah I think I had it there at first, and it goes unused, so I'll just remove it